### PR TITLE
Add Environment definition to Glossary section

### LINKED ISSUE #1729: Add virtual environment (VENV) to glossary section
### Where should the content be added?

Glossary under Miscellaneous section

### What should be added?

Meaning of virtual environment  

### Additional information

Considering the recommendation that packages be installed with conda-forge in a new environment aside from the base environment, it would be very helpful especially for a new user, to understand what a virtual environment is, making the essence of the recommendation much clearer.

### PR DESCRIPTION
**Pull request by Hil-dred**
@jakirkham The suggested correction in #1730,  from VENV to Environment has been made.

Closes #1729.

### CODE REVIEW

File: src/misc/00_intro.rst [hunk 1]
```diff
@@ -6,6 +6,9 @@ Glossary
 
 .. glossary::
 
+  ENV
```
Comment (jakirkham) [inline src/misc/00_intro.rst:9]: Would just spell this out

```suggestion
  Environment
```

**Review by jakirkham — COMMENTED** [review]
Looks good. One minor suggestion below

File: src/misc/00_intro.rst (hunk 1 above)
Comment (Hil-dred) [reply src/misc/00_intro.rst]: @jakirkham The suggested change has been made in the below commit

**Review by jakirkham — COMMENTED** [review]
Sorry had one more suggestion. Otherwise should be good.

File: src/misc/00_intro.rst [hunk 2]
```diff
@@ -6,6 +6,9 @@ Glossary
 
 .. glossary::
 
+  ENVIRONMENT
```
[the remark this answers is not in the document]
Comment (Hil-dred) [reply src/misc/00_intro.rst]: > 

@jakirkham Correction made

**Pull request merged**